irods/test/user_group_test.py

```diff
@@ -393,6 +393,8 @@ def test_group_admin_can_create_and_administer_groups__issue_426(self):
                 admin.groups.remove(lab.name)
 
     def test_group_admin_can_create_users__issue_428(self):
+        if self.sess.server_version < (4, 2, 12):
```

Comment:
That's to say, it **was** broken in 4.2 and 4.3 until we addressed it(https://github.com/irods/irods/issues/6887)

Reply:
The inequality on 396 won't be `(4,2,12) < (4,3,0)` when the server version is 4.3.0, it will be `(4,3,0) < (4,2,12)`:
```python
>>> server_version = (4,2,11)
>>> server_version < (4,2,12)
True
>>> server_version = (4,3,0)
>>> server_version < (4,2,12)
False
```
Doesn't that mean the test will not be skipped for server version 4.3.0? If it's broken in 4.3.0, then I would think that we would want to skip the test for 4.3.0. The password initialization is/will be working in 4.2.12 and 4.3.1 and beyond, right? Did I get it backwards?

Would have expected something like this:
```python
if self.sess.server_version < (4,2,12) or self.sess.server_version == (4,3,0):
```

Reply:
Done.  As they say, mind the gap....